test/acceptance/specs/RevokeApplicationAuthoritySpec.scala

```diff
@@ -80,9 +75,27 @@ class RevokeApplicationAuthoritySpec extends BaseSpec with NavigationSugar {
 
       on(AuthorizedApplicationsPage)
     }
+
+    scenario("User is not able to revoke application authority if the application is trusted") {
+      val trustedApp = someAppAuthorisation(trusted = true)
+
+      LoginStub.stubSuccessfulLogin()
+      DelegatedAuthorityStub.stubSuccessfulFetchApplicationAuthorities(Seq(trustedApp))
+      DelegatedAuthorityStub.stubSuccessfulFetchApplicationAuthority(trustedApp)
+
+      go(AuthorizedApplicationsPage)
+
+      go(WithdrawPermissionPage(trustedApp.application.id))
+
+      on(TechnicalDifficultiesPage)
+    }
```

Comment:
Is this a different behaviour than an attempt to revoke an app that doesn't exist?